index.js

```diff
@@ -38,7 +38,8 @@
           traceDebuggingMode = false,
           trackPrefix = '',
           trackRoutes = true,
-          trackUrlParams = false;
+          trackUrlParams = false,
+          readFromRoute = false;
```

Comment:
Please also add this property to the `configuration` object returned by the service. You will find it near the end of the file and it lists all of the internally defined values.